src/Apps/Sell/Routes/__tests__/ConditionRoute.jest.tsx

```diff
@@ -85,70 +97,86 @@ describe("ConditionRoute", () => {
+    renderWithRelay({ ConsignmentSubmission: () => submissionMock })
+
+    expect(screen.getByTestId("condition-input")).toHaveValue("GOOD")
+    expect(screen.getByTestId("description-input")).toHaveValue("description")
+  })
+
   describe("when form is valid", () => {
-    it("updates the submission", async () => {
-      renderWithRelay({})
+    it("updates my collection artwork", async () => {
+      renderWithRelay({ ConsignmentSubmission: () => submissionMock })
+
+      submitMutation.mockClear()
+      screen.getByText("Submit Artwork").click()
+
+      await flushPromiseQueue()
+
+      await waitFor(() => {
+        expect(submitMutation).toHaveBeenCalledWith(
+          expect.objectContaining({
+            variables: {
+              input: {
+                artworkId: "artworkId",
+                condition: "GOOD",
+                conditionDescription: "description",
+              },
+            },
+          })
+        )
+      })
     })
   })
 
   describe("when form is not valid", () => {
     it("does not update the submission", async () => {
-      renderWithRelay({})
+      renderWithRelay({
+        ConsignmentSubmission: () => submissionMockInvalid,
+      })
+
+      submitMutation.mockClear()
+      screen.getByText("Submit Artwork").click()
+
+      await flushPromiseQueue()
+
+      await waitFor(() => {
+        expect(submitMutation).not.toHaveBeenCalled()
+      })
     })
   })
 
-  describe("navigation", () => {
-    describe("in APPROVED state", () => {
-      it("saves the submission, sets the state to `SUBMITTED` & navigates to the thank you step", async () => {
-        renderWithRelay({
-          ConsignmentSubmission: () => ({
-            ...submissionMock,
-            state: "APPROVED",
-          }),
-        })
-
-        screen.getByText("Submit Artwork").click()
-
-        await waitFor(() => {
-          // expect(trackEvent).toHaveBeenCalledWith({
-          //   action: "consignmentSubmitted",
-          //   context_module: "sell",
-          //   context_owner_type: "submitArtworkStepAddPhoneNumber",
-          //   fieldsProvided: [],
-          //   submission_id: '<mock-value-for-field-"externalId">',
-          // })
-
-          expect(submitMutation).toHaveBeenCalledWith(
-            expect.objectContaining({
-              variables: {
-                input: {
-                  externalId: '<mock-value-for-field-"externalId">',
-                  title: '<mock-value-for-field-"title">',
-                },
-              },
-            })
-          )
-
-          expect(submitMutation).not.toHaveBeenCalledWith(
-            expect.objectContaining({
-              variables: {
-                input: {
-                  externalId: '<mock-value-for-field-"externalId">',
-                  state: "SUBMITTED",
-                },
-              },
-            })
-          )
+  /* describe("navigation", () => {
```

Comment:
Navigation tests are failing, having troubles figuring out why

Reply:
You need to set the submission state to "APPROVED" because this is a step that is not reachable when the submission is in state "DRAFT" or "SUBMITTED" (Tier 2 step).

[You can have a look at this test file for another additional/tier 2 route:](https://github.com/artsy/force/blob/827a09088f3df25ad3a42df553e340aed56e5a91/src/Apps/Sell/Routes/__tests__/ShippingLocationRoute.jest.tsx#L194)

Reply:
Hey @olerichter00, Conditions is the final step of tier2, so, i think, the status should be RESUBMITTED or similar (what you discussed with Devon on slack)?
So, I should probably edit the SellFlowContext file like this when we introduce a new state:
```
if (submission?.state === "DRAFT") {
      await updateSubmission({
        state: "SUBMITTED",
      })
    } else if(submission?.state === "APPROVED") {
      await updateSubmission({
        state: "RESUBMITTED",
      })
    }
```

Reply:
Maybe we could pair on the failing tests today? Would you have time?